README.md

```diff
@@ -195,13 +195,13 @@ import { BrowserRouter as Router, Route, useHistory } from 'react-router-dom';
 import Home from './Home';
 import Protected from './Protected';
 
-const oktaAuth = new OktaAuth({
-  issuer: 'https://{yourOktaDomain}.com/oauth2/default',
-  clientId: '{clientId}',
-  redirectUri: window.location.origin + '/login/callback'
-});
-
 const App = () => {
+  const oktaAuth = new OktaAuth({
+    issuer: 'https://{yourOktaDomain}.com/oauth2/default',
```

Comment:
We tend to use `https://{yourOktaDomain}/oauth2/default` on the developer blog. I think the ".com" is kinda confusing.

Reply:
That's fine with me. I just moved the config down inside the function component, but didn't touch anything else. I'll update that now.

Reply:
I believe that OktaAuth is initialized outside the component because we don't want it to follow the component lifecycle. If the component is re-rendered/mounted it may create another instance of the OktaAuth client which is generally not desirable